main.py

```diff
@@ -664,6 +664,18 @@ def get(self):
         html += ")'>Clean up duplicates of confirmed users</button>"
         self.response.write(html)
 
+    def get_db(self): # Returns the entire database as an dictionary
```

Comment:
get_db sounds like it gets a "db" object. I would call this something like read_db